### DISCUSSION TITLE
Markdown kullanım kaynakları

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus.github.io#86] [Genel]
ÖYG1 ve ÖYG2 öğrencileri, Markdown yazım kurallarını öğrenecek ve yararlandıkları kaynakları, içindeki bazı örneklerle birlikte bu başlığın altına ekleyecek.

**Comment by Sadik1Dev** [comment]
Kaynak:https://medium.com/deep-learning-turkiye/t%C3%BCrk%C3%A7e-markdown-rehberi-61779d2e2a96
Markdown düz metinleri biçimlendirmeye yarayan bir basit işaretleme dilidir.
kalın yazmak için kelimenin başına ve sonuna çift yıldız (**) konur:
**Kalın**
italik yazmak için tek yıldız (*) konur:
*eğik*
görsel eklemek için ![link adı](link adresi):
![github logo](https://upload.wikimedia.org/wikipedia/commons/thumb/9/95/Font_Awesome_5_brands_github.svg/150px-Font_Awesome_5_brands_github.svg.png)

**Comment by AhmetEmirErsoz** [comment]
Markdown ile kod eklemek oldukça kolay bunun için ters tırnak kullanıyoruz. “AltGr + virgül” kombinasyonunu kullanabilirsiniz.
`import pandas as pd`

Bir büyüktür işareti atın ve yazmaya başlayın, alıntınız hazır.
> sokratesin hayatı  sorgulatan bir alıntısı.

Kalın ve italik yazı yazmak
**bunu** istersem _yan_ yazabilirim.

Dik çizgi yapmak için “Ctrl + Alt + Tire işaret” kombinasyonunu kullanabilirsiniz.
| İsim | Yaş |
| Emir | 11 |
| Ufuk | 43 |
| Çınar | 12 |


![Referans bir isim](google.com/img.png)

Link vermekle hemen hemen aynı sadece ünlem işareti ile başlıyorsunuz.

![Markdown Logosu] (https://cdn.iconscout.com/icon/free/png-256/markdown-433293.png)

Link bırakma
[Referans bir isim](https://www.gooogle.com)
İnstagram adresime [Buradan] (instagram.com/gulen_kedy)

Son olarak Başlık Atma
AltGr + 3 = #

#Hayat büyük başlar
## Hayat orta başlar
###Hayat küçük başlar

Et (#) işareti arttıkca yazıda küçülür.

**Comment by CinarSatir** [comment]
Markdown Nedir, Ne Amaçla Kullanılır?

İşaretleme, yazılarımızı düz metin olarak yazmamıza imkan veren/ biçimlendirme dilidir . Markdown'ın temel amacı, ulaşımımızı kolay ve kolay bir şekilde ulaşımını sağlamak.

Vurgulama
**kalın** __kalın__

*italik* _italik_

**_kalın ve italik_**
uzun uzun

italik italik

ve kalın italik

Maddeleme ve Sıralama
Maddeleme:
 
KAYNAK 👍 
https://github.com/kaymal/Markdown

**Comment by CanKirkgoz** [comment]
Markdown: basit söz dizimine sahip hafif işaretleme dili. Tasarımsal olarak kolay ve çok da sade. ( "|" sembolünü yapmak için Windows'ta Ctrl+Alt+"-" tuşlarına basılır. Mac'te ise Option+"-" tuşlarına basılır.) Kalın yazı fontunda yazmak için "**" koyulur. İtalik biçimde yazmak için ise başına  "*" koyulur

Bir resim koymak için 

![Link Adı](Link Adresi) yazılır.

Kaynak: 
https://medium.com/deep-learning-turkiye/türkçe-markdown-rehberi-61779d2e2a96

**Comment by onursonkaya** [comment]
MARKDOWN NEDİR?
Dijital ortamda yazı yazmak istediğimizde (makale, blog yazısı vb.) yazının akışına uygun olarak başlık, liste, tablo gibi bazı görsel biçimleri oluşturmak için kullanılır. 
En basit tanımı ile, web yazarları için text-to-HTML (metinden HTML’e) dönüştürme aracıdır. Markdown tasarımının temel hedefi geliştiricileri tarafından okunulabilirlik (readability) olarak belirtilmektedir. Bu amaçla e-posta yazımı temelinde ağırlık göstermektedir.
Daha detaylı bir tanım olarak, düz metin biçimlendirme (markup language) sözdizimine (syntax) sahip hafif bir düz metin biçimlendirme dili (plain text formatting syntax) olarak ifade edilebilir. Aynı isimli araç sadece HTML dilini desteklese de günümüzde pek çok formatta çıktı üretebilmek için de kullanılmaktadır.
Bu bağlamda “Markdown” iki şeydir:
1.	Düz bir metin biçimlendirme sözdizimi (plain text formatting syntax),
2.	Perl ile yazılmış, düz metin biçimlendirmesini HTML’ye dönüştüren bir yazılım aracı (Markdown.pl).
Genellikle (GitHub) Readme dosyalarının oluşturulmasında kullanılsa da çevrimiçi tartışma forumlarında ve zengin metin (rich text) oluşturma araçlarında da markdown yapısından sıklıkla faydalanılmaktadır. Markdown pek çok programlama dili (R için Rmarkdown gibi), platform (Github gibi) ve framework / library (Markdig gibi) tarafından desteklenmektedir.
Markdown dosyaları genellikle *.md uzantılıdır.
Markdown Örneği
Heading Tanımları
# bölüm başlıklarını (heading; h1, h2, h3…, h6) ifade eder ve adedi kadar heading’i tanımlar.
# Heading
<h1>Heading</h1>

## Sub-heading
<h2>Sub-heading</h2>

### Sub-heading
<h3>Sub-heading</h3>
Paragraf
Bir paragraf için yeni bir satır oluşturmak yeterlidir. Her yeni satır bir paragraf olarak tanımlanır. Bir paragrafın tamamı tek bir satır olabileceği gibi, arada boş satır bırakmadan alt satırdan da devam etmek mümkündür. Paragraf tanımı Block-Level‘dır; div, pre, table gibi. Bu nedenle, yazılmadan önce bir satır boşluk (toplamda 2 ENTER) bırakılmalıdır. Span-Level (inline) elemanlar olduğu gibi yazılabilir. Line Break (<br \>) için satır sonunda 2 boşluk (space) kullanılır.
Paragraphs are separated by a blank line.
<p>Paragraphs are separated by a blank line.</p>

Two spaces at the end of a line
produces a line break.
<p>Two spaces at the end of a line<br />
produces a line break.</p>
Kalın, Eğik Yazı ve Monospace
Bir metni bold (kalın), eğik (italic) veya her ikisi ile birlikte biçimlendirmek mümkündür. Bold için metnin önüne ve ardına **, italic için _, monospace için ise ``` eklemek yeterlidir.
Text attributes _italic_,
**bold**, `monospace`.
Text attributes <em>italic</em>,
<strong>bold</strong>, <code>monospace</code>
Yatay Çizgi
Metin bölümlendirme / ayırma amacıyla kullandığımız horizontal rule için --- kullanmak yeterlidir.
---
<hr />
________________________________________
Listeler
Bullet list için * ve numbered list için rakam kullanılabilmektedir.
Bullet list:
  * apples
  * oranges
  * pears
<ul>
<li>apples</li>
<li>oranges</li>
<li>pears</li>
</ul>

Numbered list:
  1. wash
  2. rinse
  3. repeat
<ol>
<li>wash</li>
<li>rinse</li>
<li>repeat</li>
</ol>
Bağlantı
Bir metni bağlantı haline getirmek için metin köşeli parantez içerisine alınır ve yanında parantez içerisinde ilişkilendirilecek bağlantı belirtilir.
An [example](http://example.com)
An <a href="http://example.com">example</a>.
Görsel
Metin içerisinde görseller de kullanmamız mümkün. Görsel tanımı için ünlem ve köşeli parantez içerisinde görselin alt, parantez içerisinde görselin adı / yolu, tırnak içerisinde title tanımları yapılır.
![Icon-pictures.png](../markdown-nedir)
<img src="https://ceaksan.com/tr/markdown-nedir/" />
Alıntı
Bir cümleyi alıntı olarak belirtmek için cümlenin başına > koymamız yeterlidir.
> Markdown uses email-style > characters for blockquoting.
<blockquote>Markdown uses email-style &gt; characters for blockquoting.</blockquote>
Escape karakteri, \‘dir ve aşağıdaki karakterler için kullanılabilir.
•	# hash mark
•	. dot
•	+ plus sign
•	- minus sign (hyphen)
•	! exclamation mark
•	* asterisk
•	_ underscore
•	\ backslash
•	` backtick
•	{} curly braces
•	[] square brackets
•	() parentheses

https://ceaksan.com/tr/markdown-nedir

**Comment by NecipFG** [comment]
# Markdown Nedir, Neden Tercih Edilmelidir ?

Markdown, düz metin metin belgelerine biçimlendirme öğeleri eklemek için kullanabileceğiniz hafif bir biçimlendirme dilidir. 2004 yılında John Gruber tarafından oluşturulan Markdown, şu anda dünyanın en popüler biçimlendirme dillerinden biridir. Markdown'un amacı, düz metin yazılarının biçimlendirilebilmesi ve okunabilirliğinin sağlanmasıdır. Basit bir şekilde biçimlendirme öğeleri eklenebilir ve bu öğelerin hiçbiri yazı okunurken görünmez. Örneğin, başlıklar, alt başlıklar, yeni satırlar, listeler ve metin kalıpları eklenebilir. Markdown sözdizimi, okunabilirlik için tasarlanmıştır. Bu nedenle, Markdown yazılarının yazılışı, okunması kolaydır. Markdown yazılarının yazılması için biraz alıştırma gerekir, ancak bu sözdizimi kullanmaya başladıktan sonra kolayca kullanılabilir. Markdown, birçok popüler biçimlendirme dilinden daha kolaydır. Örneğin, HTML ve XML gibi diğer biçimlendirme dilleri, yazarların kodlarını yazıyor olmalarını gerektirir. Markdown sözdizimi bu kodların yazılmasını gerektirmez ve yazarların yalnızca düz metin yazmalarına izin verir. Düz metin yazıları, Markdown biçimlendirme öğeleri ile biçimlendirilebilir ve bu öğeler yazı okunurken görünmez. Ayrıca, HTML'ye dönüştürüldüğünde görünümü yaygın olarak kullanılan bir biçimde olacaktır.



### **_Maddeleme:_**
Maddelemek için, Maddeden Önce, " - " işareti koyulur.

_Örnek_ ; 
`- Madde 1`

 bize ,
- Madde 1

sonucunu verir.


### **_Vurgulama:_**

**kalın** yazı için, maddenin Başında ve Sonuna " ** " koyulur.

_Örnek_ ; `**kalın**`  , bize **kalın** sonucunu verir.

_italik_  yazısı için, maddenin Başında ve Sonuna " _ " koyulur.

_Örnek_ ; `_italik_`  , bize  _italik_  sonucunu verir.


## Bu Ödev İçin Kullandığım Kaynaklar;

[Markdown Guide](https://www.markdownguide.org/getting-started/)
[Kaymal](https://github.com/kaymal)
[SonsuzUs](https://github.com/sonsuzus)

**Comment by YagizEfeZ** [comment]
Markdown, düz metin biçimlendirme sözdizimine sahip hafif bir işaretleme dili. Tasarımı, birçok çıktı biçimine dönüştürülmesine izin verir, ancak aynı ada sahip orijinal araç yalnızca HTML'yi destekler. Markdown genellikle README dosyalarını biçimlendirmek, çevrimiçi tartışma forumlarına mesaj yazmak ve düz metin düzenleyicisi kullanarak zengin metin oluşturmak için kullanılır.

örnek: _italik_ yazı yazmak için kelimenin başına ve sonuna "_" kullanılır
**kalın** yazısını yazmak için kelimenin sonuna " ** " konulur
kaynak:https://tr.wikipedia.org

**Comment by AoiKos1243** [comment]
Markdown Nedir?

Markdown, düz-metin-biçimlendirme sözdizimine sahip hafif bir işaretleme dili. Tasarımı, birçok çıktı biçimine dönüştürülmesine izin verir, ancak aynı ada sahip orijinal araç yalnızca HTML yi destekler. Markdown genellikle BENİOKU  dosyalarını biçimlendirmek, çevrimiçi tartışma forumlarına mesaj yazmak ve düz [metin düzenleyicisi] kullanarak zengin metin oluşturmak için kullanılır. Kaynakça:[_https://tr.wikipedia.org_](https://tr.wikipedia.org)

**Örnekler**

_İtalik Yazı yazmak için başına ve sonuna_ "_" _işareti konulur._

**Kalın yazmak için ise başa ve sonra 2 yıldız (*) işareti konulur.**

> Alıntı Cümleler İçin Başına ">" işareti konulur.

- Maddeli Listeler İçin Başına "-" işareti konulur.

`Kod Yazmak için başına ve sonuna işareti  konulur:`   (`)   

- [ Görev listesi için başına (-,[) sonuna (]) gelir ] 

@sonsuzus  birini etiketlemek için başına et(@) işareti konulur.

**Comment by akgembi12** [comment]
`_italik yazı_`--->   _italik yazı_


`__kalın yazı__`--->   __kalın yazı__


`**kalın yazı**`--->   **kalın yazı**


`[Google!](google.com)` --->   [Google!](google.com)


`[Yahoo][Yahoo'nun linki]`--->[Yahoo][yahoonun linki]
`[Yahoo'nun linki]:www.yahoo.com`


[yahoonun linki]: www.yahoo.com


`![foto](https://avatars.githubusercontent.com/u/86284210?s=40&v=4)`--->![kaplan](https://avatars.githubusercontent.com/u/86284210?s=40&v=4)


`>Python programlama dili Guido Van Rossum`  
`adlı Hollandalı bir programcı tarafından 90'lı`
`yılların başında geliştirilmeye başlanmıştır.`
.|
.|
V
>Python programlama dili Guido Van Rossum
adlı Hollandalı bir programcı tarafından 90'lı
yılların başında geliştirilmeye başlanmıştır.


`* Python`
`* C`
`* C#`
`* C++`
`* Ruby`
.|
.|
V
* Python
* C
* C#
* C++
* Ruby


[Kaynak](https://www.markdowntutorial.com)

**Comment by OmerAkbayrak** [comment]
**Markdown**, yazılarımızı düz metin olarak yazmamıza imkan veren işaretleme ve biçimlendirme dilidir  Markdown'ın amacı metnimizi kolayca hazırlatıp düz metnin bile rahatça okunmasını sağlamaktır.
örneğin kod satırı yaparken;
Python Örneği:

```python

print("Merhaba")

```
yazmak için ``` kullanabiliriz

**Comment by Ersan0102** [comment]
**_Markdown dili_** yazım örnekleri
# __Başlık__
 *Markdown dilinde çok fazla başlık seçeneği vardır. Bu başlıklara erişim için başına "#" işareti konulur.*
# Başlık 1
## Başlık 2
### Başlık 3
#### Başlık 4


# __Vurgu__
*Ayrıca burada vurgu, yani fontlar da vardır. Bu fontlara da başlarına yıldız veya alt çizgi kullanılarak erişilebilir.*
**kalın** __kalın__

*italik* _italik_

**_kalın ve italik_**


# __Maddeleme ve Sıralama__

*Maddeleme listelerde, sıralamalarda, madde madde yazılan bilgi metinlerinde önemli rol oynar. Bunlara Markdown dilinde ulaşmak için yıldız veya kısa çizgi işaretlerini kullanabiliriz. Ayrıca başta kullandığınız işaret alt satıra geçtiğinizde de sizi takip eder :).*
- Madde 1
- Madde 2
- Madde 3
* Alt madde 1
* Alt madde 2
- Fıkra 1
- Fıkra 2

*Sıralama normal sayılarla yapılır. Sıralamak maddelerin -sıra açısından- derlenmesine yardımcı olur.Ayrıca başta kullandığınız sayı alt satıra geçtiğinizde de sizi ondan bir sonraki sayı olarak takip eder :).*
1. Bir
2. İki
 

# __Bağlantı ekleme__
*İnternet sitelerinde gördüğümüz bağlantıların eklenmesi için yazdığımız kodlar da Markdown dilinde bulunur.*

[Aynen böyle]*Bu size benim sevdiğim bir şarkıyı gösterir.*(https://www.youtube.com/watch?v=jvQpfppkFeg&t=199s)


# __Kod(Bloğu)__
*Kod bloğu veya kodlama dili destekli kod bloğu yaratmak için tırnak kullanılır.*

merhaba bu bir `kod` bloğu.

```
Bu da bir kod bloğu
```
*Bu da phyton örnekli bir kod*
```python
a = input("Hacim?: ")
print("Hacim şu =",a)
```

**Comment by MamiA11** [comment]
> ```python
>sayi1 = input("sayı giriniz: ")
>sayi2 = input("bir tane daha sayı giriniz: ")
>sayi1 = int(sayi1)
>sayi2 = int(sayi2)
>print("toplamı: ",sayi1 + sayi2)
>print("çarpımı: ",sayi1 * sayi2)
>print("farkı: ",sayi1 - sayi2)
>print("bölümü: ",sayi1 / sayi2)
>print("ondalıksız bölümü: ",sayi1 // sayi2)
>print("bölümünden kalanı: ",sayi1 % sayi2)

> ```
*Bu kodu* bir sayının hem **ondalıklı** hem de **ondalıksız** halleri için yazdım
***Markdown ile alakalı şeyler öğrenmek için linke tklayabilirsin.***


!(https://www.markdownguide.org/)

**Comment by miraykaracol** [comment]
Markdown, basit sözdizimine sahip hafif işaretleme dilidir. Tasarımsal olarak sade ve yazımı çok basittir.

Daha detaylı bir tanım olarak, düz metin biçimlendirme (markup language) sözdizimine (syntax) sahip hafif bir düz metin biçimlendirme dili (plain text formatting syntax) olarak ifade edilebilir.

Örnek:
`**kalın**`/ `__kalın__`= **kalın**
`*italik*` / `_italik_ `= *italik*

Link bırakma örneği:
`GitHub'a [buradan](https://github.com/) gidebilirsiniz.` = GitHub'a [buradan](https://github.com/) gidebilirsiniz.
Kaynaklar:
https://ceaksan.com/tr/markdown-nedir
https://medium.com/deep-learning-turkiye/t%C3%BCrk%C3%A7e-markdown-rehberi-61779d2e2a96

**Comment by kaanpinar04** [comment]
Kaynak: <https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet>


Inline-style: 
![alt text](https://github.com/adam-p/markdown-here/raw/master/src/common/images/icon48.png "Logo Title Text 1")

Reference-style: 
![alt text][logo]

[logo]: https://github.com/adam-p/markdown-here/raw/master/src/common/images/icon48.png "Logo Title Text 2



[I'm an inline-style link](https://www.google.com)

[I'm an inline-style link with title](https://www.google.com "Google's Homepage")

[I'm a reference-style link][Arbitrary case-insensitive reference text]

[I'm a relative reference to a repository file](../blob/master/LICENSE)

[You can use numbers for reference-style link definitions][1]

Or leave it empty and use the [link text itself].


[^1]: My reference.
[^2]: Every new line should be prefixed with 2 spaces.  
  This allows you to have a footnote with multiple lines.
[^note]:

**Comment by Atakanoyg1** [comment]
Markdown da tıpkı HTML gibi bir etiketleme dili. Yani yazdığınız yazının bir yerlerine özel etiketler bırakıyorsunuz, yazının son halini gördüğünüz program da o etiketlere bakarak yazının nasıl gözükeceğine karar veriyor. Bu program herhangi bir kelime işlemcisi ya da tarayıcı olabilir. Bu dili kullanabilmek için herhangi bir yere .md veya .markdown uzantılı dosya oluşturmanız yeterli. Daha sonra onu dilediğiniz yazı editörü ile açabilirsiniz.

*italik*
**kalın**hazırlanmış
***kalın italik***
~~çizik~~

kaynakca:
https://www.technopat.net/2019/12/09/markdown-ile-yazi-yazma-rehberi/

**Comment by OmerB2** [comment]
Markdown, John Gruber ve Aaron Swartz tarafından 2004 yılında geliştirilmiş ve bu yana github, forum ve sitelerde kullanılan zengin bir işaretleme dilidir. HTML destekli bu işaretleme dili ile zengin metinler oluşturabilirsiniz. Markdown uzantısı .md ve .markdown olarak bilinir.

Başlık Ekleme
Oluşturacağımız metinlerin olmazsa olmazı olan başlıklar aşağıdaki yöntemler ile oluşturabilirsiniz. Başlıklarda önemli olan tek nokta # işaretinin sayısıdır. Ne kadar fazla # işareti var ise başlığın boyutu o kadar küçülecektir.

# Başlık1 
## Başlık2
### Başlık3

Linkli Başlık eklemek içinde önce köşeli parantez içine metnimizi sonra tırnak paranterz içine link adresini eklememiz yeterli.

# [Başlık](https://www.yazilimturkiye.com/)

Resim Ekleme
Resim eklemek için birden fazla yöntem bulunmakta, aşağıda en sık kullanılan 2 yöntem gösterilmekte.

Birinci yöntem; markdown dilini kullanarak görsel ekleme.

![]( https://upload.wikimedia.org/wikipedia/commons/thumb/4/48/Markdown-mark.svg/1200px-Markdown-mark.svg.png)

İkinci yöntem; HTML kullanarak resim ekleme. Bu yöntemde resmi hem ortalamış hemde boyutunu kendimiz ayarlamış olduk, yani kontrol bizde.

<p align="center">
  <img width="250" height="250" src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/48/Markdown-mark.svg/1200px-Markdown-mark.svg.png">
</p>

Liste Oluşturmak
Liste oluşturmak tıpkı HTML işaretleme dilinde olduğu gibi sıralı ve sırasız şeklinde 2 tiptir. Burada önemli olan başlarına koyacağınız işarettir.

Sıralı Liste; oluşturmak için metinin başına 1., 2., 3., gibi düzenli olarak sayıları sıralamamız yeterli.

1. Liste
2. Liste
3. Liste

Sırasız Liste; oluşturmak için tek yapmamız gereken listemizin başına + işaretini koymaktır.

+ Liste 1
+ Liste 2
+ Liste 3
  + Liste 3.1
  + Listes 3.2

Sırasız Liste 2; burada da tıpkı diğer örnekteki gibi tek yapmamız gereken başına – işaretini koymaktır. Genelde en çok kullanılan liste tipi budur.

- Liste 1
- Liste 2
- Liste 3
    - Liste 3.1
    - Liste 3.2

Link Ekleme
Markdown ile neredeyse her şeye link verebilirsiniz. Link verir iken [] köşeli parantez içine metnimizi daha sonrada tırnak parantez () içine de link adresi verilir. Link vermek için yapmanız gerekenler aşağıdadır.

[YazılımTürkiye](https://www.yazilimturkiye.com/)

Kod Satırı Ekleme
Metinlerinize kod kutucuğu eklemek için ters virgül kullanabilirsiniz. Kodlarınızı Ters virgül içine yazmanız gerekmekte. Ters virgül yapmak için AltGR + Virgül tuşlarına basınız. Aşağıdaki örneği kullanabilirsiniz.

` print ('Hello World') `

Çoklu satırda kod ekleme; bunun için ilk örneğimizdeki gibi ters virgül kullanıyoruz fakat bu sefer ters virgül sayısı 3 olmalıdır.

```<!DOCTYPE html>
<html>
    <head>

        <title>Hello world!</title>
    </head>
    <body>
        <h1>Hello world!</h1>
    </body>
</html>
```
Tablo Oluşturma
Markdown ile tablo oluşturmak hem çok zevkli hemde çok kolaydır. Tek yapmanız gereken sutunlar için düz çizgi satırlar için ise tire koymanız yeterlidir. Düz çizgi yapmak için Ctrl + Alt + Tire İşareti kompinasyonunu yapınız. Örnek Tablo aşağıdadır.


| Birinci Başlık| İkinci Başlık    |
| ------------- | ---------------- |
| İçerik        | İçerik           |
| İçerik        | İçerik           |

Ayraçlar
Seffat bir ayraç koymak metinlerinizin okunabilirliğini ve içeriklerin ayrılması için çok önemli olabilir. Ayraç yapmak için 3 adet tire koymanı yeterli.

---
Alıntılama
Alıntı yapmak için klavyenizdeki büyüktür işaretiniz koymanız ve metnininizi yazmanız yeterli.

> Ne demişler? çalışıyorsa dokunma...
Seçim Kutucukları Oluşturma
Metninizde seçim kutucukları (checkbox) kullanmak istiyor iseniz aşağıdaki yöntemler ile ekleyebilirsiniz. Burada önemli olan köşeli parantezin içine bir boşluk koymanız. Dilersenizde seçim kutularından birinin içine koyacağınız bir X karakteri ile seçili olarak gösterebilirsiniz.

[x] Masaüstü
[ ] Web
[ ] Mobil
[ ] Yapay Zeka

Metinsel Geliştirmeler
Markdown kullanır iken metinlerinizi ister kalın istersenizde italik yapabilirsiniz.

**kalın yazma**
_italik_


Kaynak: (https://www.yazilimturkiye.com/markdown-nedir-turkce-kullanim-rehberi/)

**Comment by D3r3n** [comment]
1) Düz bir metin biçimlendirme sözdizimi dir
2) Perl ile yazılmış bir metin biçimlendirmesini HTML'ye dönüştüren bir yazılım aracıdır

Markdown dosyaları genellikle ".md" uzantılıdır

•Heading Tanımlar
# bölüm başlıkları (heading h1, h2, h3...) 
## bölüm başlıkları
### bölüm başlıkları 

•Her paragraf için yeni bir bölüm oluşturmak yeterlidir

•Bold Metin için **..... **
•İtalik için _... _
•Monospare için """ eklemek yeterlidir

•Alıntı için büyüktür işareti (>) 

•Escape karakteri \'dir


Kaynak:
https://ceaksan.com/tr/markdown-nedir

**Comment by TahaGuzell** [comment]
Markdown , herhangi bir cihazda harika görünen metni biçimlendirmenin basit bir yoludur. Yazı tipi boyutunu, rengini veya türünü değiştirmek gibi süslü bir şey yapmaz - yalnızca zaten bildiğiniz klavye sembollerini kullanarak gerekli olanları yapar.

Tip	Veya	… almak
*İtalik*	_İtalik_	İtalik
**Gözü pek**	__Gözü pek__	Gözü pek
# Başlık 1	Başlık 1
=========	
Başlık 1
## Başlık 2	Başlık 2
---------	
Başlık 2
[Bağlantı](http://a.com)	[Link][1]
⋮
[1]: http://b.org	[Bağlantı](https://commonmark.org/)
![Resim](http://url/a.png)	![Resim][1]
⋮
[1]: http://url/b.jpg	indirim
> Blok alıntı	 	
blok alıntı
* Liste
* Liste
* Liste

- Liste
- Liste
- Liste

Liste
Liste
Liste

1. Bir
2. İki
3. Üç

1) Bir
2) İki
3) Üç

Bir
İki
Üç
Yatay kural:

---	Yatay kural:

***	Yatay kural:
Backticks ile "Satır içi kod"	 	Inline codegeri tepme ile
```
# kod bloğu
yazdır '3 backticks veya'
print '4 boşluk girintisi'
```	···· # kod bloğu
···· '3 backticks veya' yazdırın
···· '4 boşluk girintisi ' yazdırın	
# kod bloğu
yazdır '3 geri tepme veya'
yazdır '4 boşluk girintisi'

Kaynak:
Tip	Veya	… almak
*İtalik*	_İtalik_	İtalik
**Gözü pek**	__Gözü pek__	Gözü pek
# Başlık 1	Başlık 1
=========	
Başlık 1
## Başlık 2	Başlık 2
---------	
Başlık 2
[Bağlantı](http://a.com)	[Link][1]
⋮
[1]: http://b.org	[Bağlantı](https://commonmark.org/)
![Resim](http://url/a.png)	![Resim][1]
⋮
[1]: http://url/b.jpg	indirim
> Blok alıntı	 	
blok alıntı
* Liste
* Liste
* Liste

- Liste
- Liste
- Liste
Liste
Liste
Liste
1. Bir
2. İki
3. Üç

1) Bir
2) İki
3) Üç

Bir
İki
Üç
Yatay kural:

---	Yatay kural:

***	Yatay kural:
Backticks ile "Satır içi kod"	 	Inline codegeri tepme ile
```
# kod bloğu
yazdır '3 backticks veya'
print '4 boşluk girintisi'
```	···· # kod bloğu
···· '3 backticks veya' yazdırın
···· '4 boşluk girintisi ' yazdırın	
# kod bloğu
yazdır '3 geri tepme veya'
yazdır '4 boşluk girintisi'



Kaynak:
https://commonmark.org/help/





































































































[](url)

**Comment by dudusoo** [comment]
*Markdown, kolay anlaşılabilen bir işaretleme dilidir. Tasarımı, birçok çıktı biçimine dönüştürülmesine izin verir.

Örneğin Markdown sayesinde;
"#### Tablo Elemanı 
| Ad | Soyad |
|--|--| 
|Kozmocan| Uzayaçıkan |" şeklinde bir kullanımda anlaşılır bir görüntü elde edebiliriz.

**Comment by Barlasgng** [comment]
Markdown düz metinleri biçimlendirmeye yarayan bir basit işaretleme dilidir.
kalın yazmak için kelimenin başına ve sonuna çift yıldız (**) konur:
Kalın
italik yazmak için tek yıldız (*) konur:
eğik
görsel eklemek için ![link adı](link adresi):

**Comment by adasue** [comment]
Markdown, düz metin metin belgelerine biçimlendirme öğeleri eklemek için kullanabileceğiniz hafif bir biçimlendirme dilidir. 2004 yılında John Gruber tarafından oluşturulan Markdown, şu anda dünyanın en popüler biçimlendirme dillerinden biridir.
Kalın, Eğik, Üstü çizili ve Tek Satırlı Kod

Kalın yapmak için **bu yazı kalın olacak** , eğik yapmak için _bu yazı eğik olacak_ , üstü çizik yapmak için ~~buranın üstü çizik olacak~~ ve tek satır kod için `kod buraya` şeklinde kullanılabilir.

Kaynakça: (https://www.markdownguide.org/)
